SpatialGDK/Source/SpatialGDK/SpatialGDK.Build.cs

```diff
@@ -142,13 +123,33 @@ public SpatialGDK(ReadOnlyTargetRules Target) : base(Target)
             WorkerImportLib = Path.Combine(WorkerLibraryDir, WorkerImportLib);
             PublicRuntimeLibraryPaths.Add(WorkerLibraryDir);
 
+            PublicAdditionalLibraries.Add(WorkerImportLib);
         }
         else
         {
-            PublicLibraryPaths.AddRange(WorkerLibraryPaths);
-        }
+            WorkerLibraryPaths.Clear();
```

Comment:
This one isn't actually used before, so can remove the original definition and make this local to this scope.